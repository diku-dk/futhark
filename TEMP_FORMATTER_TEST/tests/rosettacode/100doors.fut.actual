25a71d30cc270d6d64f488d888c2c994
