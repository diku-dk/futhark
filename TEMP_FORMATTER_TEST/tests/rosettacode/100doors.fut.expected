875edd811729bbf40285cf7cfcb00cb7
